tests/utils/test_async_logging_queue.py

```diff
@@ -43,6 +44,46 @@ def consume_queue_data(self, run_id, metrics, tags, params):
 
 
 def test_single_thread_publish_consume_queue():
+    from mlflow.environment_variables import MLFLOW_ASYNC_LOGGING_WAITING_TIME
+
+    MLFLOW_ASYNC_LOGGING_WAITING_TIME.set("3")
+
+    with patch.object(
+        AsyncLoggingQueue, "_batch_logging_worker_threadpool", create=True
+    ) as mock_worker_threadpool, patch.object(
+        AsyncLoggingQueue, "_batch_status_check_threadpool", create=True
+    ) as mock_check_threadpool:
+        mock_worker_threadpool.submit = MagicMock()
+        mock_check_threadpool.submit = MagicMock()
+        mock_worker_threadpool.shutdown = MagicMock()
+        mock_check_threadpool.shutdown = MagicMock()
+
+        run_id = "test_run_id"
+        run_data = RunData()
+        async_logging_queue = AsyncLoggingQueue(run_data.consume_queue_data)
+        async_logging_queue.activate()
+        async_logging_queue._batch_logging_worker_threadpool = mock_worker_threadpool
+        async_logging_queue._batch_status_check_threadpool = mock_check_threadpool
+        metrics_sent = []
```

Comment:
Are these contents supposed to be validated?

Reply:
that's noise I left behind after copy paste, deleted.